terraso_backend/apps/core/static/instance_restore.js

```diff
@@ -0,0 +1,74 @@
+(function () {
+
+  const button = document.getElementById("restore-button");
+  const statusP = document.getElementById("restore-status");
+
+  function getCookie(name) {
+    let cookieValue = null;
+    if (document.cookie && document.cookie !== '') {
```

Comment:
Won't `if (document.cookie)` do the same thing?

Reply:
I think so, but I copied this function directly from the [Django docs](https://docs.djangoproject.com/en/4.1/howto/csrf/#acquiring-the-token-if-csrf-use-sessions-and-csrf-cookie-httponly-are-false) so I'm not sure if there's a corner case that the double check covers.